Cargo.lock

```diff
@@ -11,15 +11,15 @@ dependencies = [
  "crc 1.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "fs2 0.4.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "getopts 0.2.14 (registry+https://github.com/rust-lang/crates.io-index)",
- "kvproto 0.0.1 (git+https://github.com/pingcap/kvproto)",
+ "kvproto 0.0.1 (git+https://github.com/pingcap/kvproto?branch=siddontang/protobuf-1.2.1)",
```

Comment:
kvproto must use 1.2.1, so here I have to use the branch now. 

After kvproto and tipb PR merged, I will use the master branch.